### ISSUE TITLE
Niconico error getting video title

### ISSUE THREAD

**Issue by Konosprod** [soimort/you-get#2160]
Hello,

since few days, actually something like 3 or 4 days, I'm unable to download any video from NicoNico, here is the stacktrace.

```
Logging in...
[DEBUG] get_response: http://www.nicovideo.jp/watch/sm31589624
you-get: version 0.4.775, a tiny downloader that scrapes the web.
you-get: ['http://www.nicovideo.jp/watch/sm31589624']
Traceback (most recent call last):
  File "./you-get", line 11, in <module>
    you_get.main(repo_path=_filepath)
  File "/var/www/test/you-get/src/you_get/__main__.py", line 92, in main
    main(**kwargs)
  File "/var/www/test/you-get/src/you_get/common.py", line 1468, in main
    script_main('you-get', any_download, any_download_playlist, **kwargs)
  File "/var/www/test/you-get/src/you_get/common.py", line 1379, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only, json_output=json_output, caption=caption)
  File "/var/www/test/you-get/src/you_get/common.py", line 1181, in download_main
    download(url, **kwargs)
  File "/var/www/test/you-get/src/you_get/common.py", line 1461, in any_download
    m.download(url, **kwargs)
  File "/var/www/test/you-get/src/you_get/extractors/nicovideo.py", line 34, in nicovideo_download
    title = unicodize(r1(r'<span class="videoHeaderTitle"[^>]*>([^<]+)</span>', html))
  File "/var/www/test/you-get/src/you_get/common.py", line 255, in unicodize
    return re.sub(r'\\u([0-9A-Fa-f][0-9A-Fa-f][0-9A-Fa-f][0-9A-Fa-f])', lambda x: chr(int(x.group(0)[2:], 16)), text)
  File "/usr/lib/python3.5/re.py", line 182, in sub
    return _compile(pattern, flags).sub(repl, string, count)
TypeError: expected string or bytes-like object

```
I did try to watch the video in my browser, I tried to update my you-get with pip, and tried the develop version, both did the same thing.

I guess it might be a change in the html  code in a page of a video on NND.

Thank you for reading me.

**Comment by rosynirvana** [comment]
this should be fixed with 0f839ba
```
[DEBUG] get_response: http://www.nicovideo.jp/watch/sm31589624
[DEBUG] get_response: http://flapi.nicovideo.jp/api/getflv?v=sm31589624
[DEBUG] url_info: http://smile-fnl31.nicovideo.jp/smile?m=31589624.19720
Site:       Nicovideo.jp
Title:      「Blue Symphony」踊ってみた - ニコニコ動画
Type:       MPEG-4 video (video/mp4)
Size:       18.38 MiB (19272529 Bytes)

Downloading 「Blue Symphony」踊ってみた - ニコニコ動画.mp4 ...
 100% ( 18.4/ 18.4MB) ├█████████████████████████████████████████┤[1/1]  159 kB/s
```

```
[DEBUG] get_response: http://www.nicovideo.jp/watch/sm31244573
[DEBUG] get_response: http://flapi.nicovideo.jp/api/getflv?v=sm31244573
[DEBUG] url_info: http://smile-fnl60.nicovideo.jp/smile?m=31244573.59923
Site:       Nicovideo.jp
Title:      【トレス】MUSIC♪【ミリマス】 - ニコニコ動画
Type:       MPEG-4 video (video/mp4)
Size:       70.77 MiB (74211301 Bytes)

Downloading 【トレス】MUSIC♪【ミリマス】 - ニコニコ動画.mp4 ...
35.7% ( 25.2/ 70.8MB) ├███████████████──────────────────────────┤[1/1]  139 kB/s
```

or perhaps you will prefer to remove the ``- ニコニコ動画'' suffix?

**Comment by Konosprod** [comment]
It works, thank you.

I guess you should remove it, because it's not the real title of the video.
